src/components/dropdown-filter-ajax/dropdown-filter-ajax.js

```diff
@@ -198,6 +221,10 @@ class DropdownFilterAjax extends DropdownFilter {
         filter: this.props.create ? prevState.filter : null
       }));
 
+      if (this.pendingRequest !== null) {
```

Comment:
should this also clear the timeout if there is one?

Reply:
Yea makes sense to add it also here